### PR TITLE
Rounding numbers in dataframe outputs.

### PR DESCRIPTION
**Pull request by amedumer**
## Summary:
This PR introduces a variable for setting decimals in dataframe outputs and adds a percentage sign for one of the dataframes.

## Changelog:
[SDK] [ADDED] - decimals variable for dataframe outputs
[SDK] [CHANGED] - customer percentage column in plot_paying_customers_flow function now displays percentage.

## Test Plan:
run notebook

### CODE REVIEW

File: src/exploratory.py [hunk 1]
```diff
@@ -39,13 +39,15 @@ def __init__(
         event_name_col: str = "event_name",
         value_col: str = "purchase_value",
         segment_feature_cols: List[str] = None,
+        decimals: int = 1,
```
Comment (mikaelwallgren) [inline src/exploratory.py:42]: nit: could decimalPlaces or roundingPrecision be a better name for this?

File: src/exploratory.py [hunk 2]
```diff
@@ -883,7 +891,7 @@ def estimate_ltv_impact(
         """
         print(output_txt)
 
-        return data
+        return data.round(self.decimals)
```
Comment (mikaelwallgren) [inline src/exploratory.py:894]: Is there any case where a user doesn't want to have any rounding? A work around can be to add a large enough number as input to the param, but another solution could be to allow null and don't apply rounding. (only if there is a use case for it)

**Review by mikaelwallgren — APPROVED** [review]
LGTM, please see comments and nit. Feel free to skip them if you don't think it's needed.

File: src/exploratory.py (hunk 1 above)
Comment (amedumer) [reply src/exploratory.py:42]: great point, changing it now

**Pull request merged**